python/dgl/distributed/dist_graph.py

```diff
@@ -453,6 +454,25 @@ def rank(self):
             client_id_in_part = rpc.get_rank() % num_client_per_part
             return int(self._gpb.partid * num_client_per_part + client_id_in_part)
 
+    def find_edges(self, edges):
+        """ Given an edge ID array, return the source
+        and destination node ID array ``s`` and ``d``.  ``s[i]`` and ``d[i]``
+        are source and destination node ID for edge ``eid[i]``.
+
+        Parameters
+        ----------
+        edges : tensor
+            The edge ID array.
+
+        Returns
+        -------
+        tensor
+            The source node ID array.
+        tensor
+            The destination node ID array.
+        """
+        return find_edges(self, edges)
```

Comment:
you can rename the one imported graph_services with another name? it looks like recursive call.

Reply:
also please provide a unit test for this operation.